core/templates/dev/head/domain/exploration/ExplorationObjectFactory.js

```diff
@@ -76,6 +76,12 @@ oppia.factory('ExplorationObjectFactory', [
       return interactionId ? INTERACTION_SPECS[interactionId].instructions : '';
     };
 
+    Exploration.prototype.getNarrowInstructions = function(stateName) {
+      var interactionId = this.getInteractionId(stateName);
+      var narrowInstructions = INTERACTION_SPECS[interactionId];
```

Comment:
Ah, for getting around the linter, I suggest:

    return (
      interactionId ?
      INTERACTION_SPECS[interactionId].narrow_instructions :
      '');